options.go

```diff
@@ -21,6 +21,10 @@ type Options struct {
 	// By default os.TempDir() is used.
 	WorkDir string
 
+	// Keep temporary files until Close.
+	// Default: Immediately remove temporary files.
+	KeepFiles bool
```

Comment:
I don't think this option makes sense, I prefer not to add it and always remove immediately

Reply:
This was to keep your test that checks for compressed temporary files. Do you have a simple way to get around that?